### PR TITLE
Login query

### PR DESCRIPTION
**Pull request by djavanm**
Allow user input for logging in.

### CODE REVIEW

File: src/util/apiCalls.js [hunk 1]
```diff
@@ -79,6 +79,37 @@ export const userQuery = gql`
   }
   `;
 
+  export const loginQuery = gql`
```
Comment (Myrdden) [inline src/util/apiCalls.js]:
```suggestion
  export const LOGIN = gql`
```

File: src/components/LoginForm/LoginForm.js [hunk 2]
```diff
@@ -1,35 +1,30 @@
 import React, { useState, useContext } from 'react';
-import { InterviewsContext, UserContext, QuestionsContext } from '../../Context';
-import { login, getInterviews, getQuestions, userQuery } from '../../util/apiCalls';
-import gql from 'graphql-tag';
+import { UserContext } from '../../Context';
+import { loginQuery } from '../../util/apiCalls';
```
Comment (Myrdden) [inline src/components/LoginForm/LoginForm.js]:
```suggestion
import { LOGIN } from '../../util/apiCalls';
```

File: src/components/LoginForm/LoginForm.js [hunk 3]
```diff
@@ -1,35 +1,30 @@
...
 
 export const LoginForm = () => {
-  const { setQuestions } = useContext(QuestionsContext)
-  const { setInterviews } = useContext(InterviewsContext);
   const { setUser } = useContext(UserContext);
   const [emailInput, setEmailInput] = useState('');
   const [passwordInput, setPasswordInput] = useState('');
 
-  const [getUser, { loading, error, data }] = useLazyQuery(userQuery);
+  const [loginUser, { loading, error, data }] = useLazyQuery(loginQuery);
```
Comment (Myrdden) [inline src/components/LoginForm/LoginForm.js]:
```suggestion
  const [loginUser, { loading, error, data }] = useLazyQuery(LOGIN);
```

**Review by Myrdden — CHANGES_REQUESTED** [review]
just convention lol

**Review by Myrdden — APPROVED** [review]

**Comment by eoneill23** [comment]
Looks great!

**Pull request merged**